taskcluster/ci/firebase-performance/kind.yml

```diff
@@ -0,0 +1,23 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+---
+loader: taskgraph.loader.transform:loader
+
+transforms:
+    - ffios_taskgraph.transforms.bitrise:transforms
+    - ffios_taskgraph.transforms.secrets:transforms
+    - taskgraph.transforms.job:transforms
+    - taskgraph.transforms.task:transforms
+
+
+tasks:
+    tests:
+        bitrise-workflow: Firebase_Performance_Test
+        description: Run performance tests in Firebase on physical devices
+        run-on-tasks-for: []
+        treeherder:
+            symbol: firebase-perf
+            kind: test
+            platform: ios/opt
```

Comment:
@jjSDET my understanding after yesterday's meeting is that we should change this so that we can differentiate the tests coming from the sim and the physical devices...

Reply:
ah, yes. i think that's all we needed up update. got that in. hopefully that should be it for now and we can see how it looks in perfherder. might want to tweak some stuff afterwards, but forget what that might be until then.